temporal_capi/src/plain_month_day.rs

```diff
@@ -99,5 +99,15 @@ pub mod ffi {
                 .map(|x| Box::new(PlainDate(x)))
                 .map_err(Into::into)
         }
+        pub fn to_ixdtf_string(
+            &self,
+            display_calendar: DisplayCalendar,
+            write: &mut DiplomatWrite,
+        ) {
+            // TODO this double-allocates, an API returning a Writeable or impl Write would be better
```

Comment:
note: definitely intrigued about this

Reply:
Yes I was wondering about that too.

apart from that, LGTM

Reply:
https://github.com/boa-dev/temporal/pull/340 is some of the types.